src/main/flight/pid.h

```diff
@@ -45,6 +45,7 @@ FP-PID has been rescaled to match LuxFloat (and MWRewrite) from Cleanflight 1.13
 #define FP_PID_RATE_P_MULTIPLIER    31.0f
 #define FP_PID_RATE_I_MULTIPLIER    4.0f
 #define FP_PID_RATE_D_MULTIPLIER    1905.0f
+#define FP_PID_RATE_D_FF_MULTIPLIER   7270.0f
```

Comment:
Would it make sense to have the same multiplier as D-term? It's effectively the same thing, wouldn't it make sense to use the same scale?

Reply:
I want to keep this thing 1:1 on values with Betaflight. That will be simpler to migrate from BF to INAV

Reply:
Makes sense then.